app/Models/Logo.php

```diff
@@ -43,15 +43,15 @@ class Logo extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
         return $this->belongsTo('App\Models\User');
     }
 
-    public function logovote(){
+    public function logovote() {
         return $this->belongsToMany('App\Models\LogoVote');
     }
 
-    public function voteresult(){
+    public function voteresult() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->